crates/subspace-farmer/src/single_plot_farm.rs

```diff
@@ -390,15 +390,15 @@ impl SinglePlotFarm {
 
         // Start the farming task
         let farming = enable_farming.then(|| {
-            Farming::start(
+            Handle::current().block_on(Farming::start(
```

Comment:
Is there a difference in `Handle::current().block_on()` and `tokio::task::spawn_blocking`? :thinking:

Reply:
They are completely different things. `Handle::current().block_on()` blocks current thread until future resolves, `tokio::task::spawn_blocking()` spawns blocking task on thread pool and gives you a handle back that you may wait for if you want. Here we want to wait for farming to start before moving any further.